forage-android/src/main/java/com/joinforage/forage/android/core/element/ElementValidationErrors.kt

```diff
@@ -20,9 +38,17 @@ val TooLongEbtPanError = ElementValidationError(
 )
 
 // PIN Input Errors
+/**
+ * A type of [ElementValidationError] that is thrown when a customer submits an incomplete
+ * EBT Card PIN.
+ */
 val IncompleteEbtPinError = ElementValidationError(
     detail = "Your EBT card PIN is incomplete."
 )
+
+/**
+ * A type of [ElementValidationError] that is thrown when a customer submits an invalid EBT Card PIN.
```

Comment:
the errors in this file are not exactly "thrown", but rather "set" (the `state.validationError` is set with the `ElementValidationError`)

Reply:
Thanks for syncing IRL that I can just say occurs 💐